### PR TITLE
18.10.0 Release Notes

### PR DESCRIPTION
**Pull request by luciferous**
Also, small formatting changes for Fall Review.

### CODE REVIEW

File: source/blog/2018-10-24-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+* finatra-http, finatra-thrift: Make HTTP and Thrift StatsFitlers "Response Classification"
+  aware. [4085d40c](https://github.com/twitter/finatra/commit/4085d40cb7cb607101564bbc488f9a1daea56dff)
+
+* finatra-http, finatra-thrift: (BREAKING API CHANGE) Update the `DarkTrafficFilterModule` in
+  both HTTP and Thrift to allow for specifying further configuration of the underlying Finagle client.
+  This allows users the ability to set Finagle client concerns like ResponseClassification or other
+  configuration not expressed by the DarkTrafficFilterModule's API.
+
+  - Additionally, the Thrift `DarkTrafficFilterModule` has been updated to be ThriftMux only. For more information on mux see: [What is ThriftMux](https://twitter.github.io/finagle/guide/FAQ.html?highlight=thriftmux#what-is-thriftmux>).
```
Comment (stefanlance) [inline source/blog/2018-10-24-release-notes.md]: I think the `>` should be dropped from the section name. As is, it doesn't link to the "What is ThriftMux?" section.

**Comment by stefanlance** [comment]
Also, why was the `publish` submodule changed?

File: source/blog/2018-10-24-release-notes.md (hunk 1 above)
Comment (luciferous) [reply source/blog/2018-10-24-release-notes.md]:
```suggestion
  - Additionally, the Thrift `DarkTrafficFilterModule` has been updated to be ThriftMux only. For more information on mux see: [What is ThriftMux](https://twitter.github.io/finagle/guide/FAQ.html?highlight=thriftmux#what-is-thriftmux).
```

**Comment by luciferous** [comment]
@stefanlance not sure about why publish was changed, I'll reset it

**Pull request merged**